src/components/AppProvider/AppProvider.scss

```diff
@@ -78,7 +78,7 @@ h6,
 p {
   margin: 0;
   font-size: 1em;
-  font-weight: 400;
+  font-weight: var(--p-font-weight-regular);
```

Comment:
The only thing I am unsure about is if we want to continue breaking apart variables `fontWeightRegular` as `font-weight-regular`. It would be good to quickly get some opinions before shipping. I personally would probably prefer less dashes but they are easy to find with the matching `font-weight` value.

Reply:
I'll create a separate issue for this!